sentences/pl/cover_HassTurnOff.yaml

```diff
@@ -2,41 +2,246 @@ language: pl
+          - "<close> [<area>] [<all>] drzwi"
+          - "[<area>] <close> [<all>] drzwi"
+        slots:
+          domain: cover
+          device_class: door
+        response: "cover_door"
+
+      # ---------- Damper ----------
+      - sentences:
+          - "(<close>|opuść) <name> [<area>]"
+          - "(<close>|opuść) [<area>] <name>"
+          - "[<area>] (<close>|opuść) <name>"
+        requires_context:
+          domain: cover
+          device_class: damper
+        response: "cover_damper"
+
+      - sentences:
+          - "(<close>|opuść) [<all>] klapy [<area>]"
+          - "(<close>|opuść) [<area>] [<all>] klapy"
+          - "[<area>] (<close>|opuść) [<all>] klapy"
+        slots:
+          domain: cover
+          device_class: damper
+        response: "cover_dampers"
+
+      # ---------- Awning ----------
+      - sentences:
+          - "(<close>|<close_blind>) <name> [<area>]"
+          - "(<close>|<close_blind>) [<area>] <name>"
+          - "[<area>] (<close>|<close_blind>) <name>"
+        requires_context:
+          domain: cover
+          device_class: awning
+        response: "cover_awning"
+
+      - sentences:
+          - "(<close>|<close_blind>) [<all>] markizy [<area>]"
+          - "(<close>|<close_blind>) [<area>] [<all>] markizy"
+          - "[<area>] (<close>|<close_blind>) [<all>] markizy"
+        slots:
+          domain: cover
+          device_class: awning
+        response: "cover_awnings"
 
+      # ---------- Window ----------
       - sentences:
-          - "(<close>|<close_blind>) [<all>] rolety"
+          - "<close> <name> [<area>]"
+          - "<close> [<area>] <name>"
+          - "[<area>] <close> <name>"
+        requires_context:
+          domain: cover
+          device_class: window
+        response: "cover_window"
+
+      - sentences:
+          - "<close> [<all>] okna [<area>]"
+          - "<close> [<area>] [<all>] okna"
+          - "[<area>] <close> [<all>] okna"
         slots:
           domain: cover
-          # device_class: curtain
+          device_class: window
+        response: "cover_windows"
+
+      # ---------- Other ----------
+      - sentences:
+          - "<close> <name> [<area>]"
+          - "<close> [<area>] <name>"
+          - "[<area>] <close> <name>"
+        requires_context:
+          domain: cover
+        excludes_context:
           device_class:
-            - curtain
+            - blind
             - shutter
+            - shade
             - curtain
-        response: cover_blind
+            - garage
+            - gate
+            - door
+            - damper
+            - awning
+            - window
+        response: "cover"
+
+      # TODO: Assist don't support translations for response for use {{ slots.device_class }} {{ state.domain }}. We have two way to correct support polish responses:
+      # 1. Create seperate sentences for each device_class or domens (this was use too choosed)
```

Comment:
I know this is a comment and all but this line alone has like at least 3 errors, please correct just for my peace of mind, thank you very much 😁